### DISCUSSION TITLE
"Copy to clipboard" button

### DISCUSSION THREAD

**Discussion by nitefood** [GyulyVGC/sniffnet#207] [Ideas]
In the _Inspect_ tab, it would be very useful to have a way to quickly copy SRC or DST IP address to clipboard. Otherwise, if it doesn't match your design, I guess it could be added to the _Connection details_ popup. Either way, I think it could be a helpful addition.

Thanks for the awesome project, ottimo lavoro! 💪

**Comment by GyulyVGC** [comment]
Thank you very much (grazie 🇮🇹)!

As stated in https://github.com/GyulyVGC/sniffnet/issues/32, the [GUI library (Iced)](https://github.com/iced-rs/iced) I depend on currently does not support text selection yet.

Adding a copy button could be doable but it's not planned because Iced is currently working on text improvements and will support text selection sooner or later.

However I totally agree with you, and I'm so happy you like Sniffnet 💖

**Comment by GyulyVGC** [comment]
Since this feature has been requested again by another user in #406, it could be worth to implement it, also considered that Iced is still far from supporting text selection.

I'm closing this discussion in favour of issue #406

**Comment by GyulyVGC** [comment]
Implemented in #409.
Feedback's welcome 😄
